dns_check/datadog_checks/dns_check/dns_check.py

```diff
@@ -78,7 +79,10 @@ def check(self, _):
                     raise AssertionError("Expected an NXDOMAIN, got a result.")
             else:
                 answer = resolver.query(self.hostname, rdtype=self.record_type)  # dns.resolver.Answer
-                assert answer.rrset.items[0].to_text()
+
+                items = list(answer.rrset.items.keys()) if PY3 else answer.rrset.items
+                assert items[0].to_text()
```

Comment:
I wonder if what was originally intended was something like

```suggestion
                assert any(it.to_text() for it in items)
```

That would make it independent of whether it's a list or a dict, and would seem easier to understand (otherwise, some comment explaining why the *first* element is so important here would be in order).

Reply:
I don't have the background, let's do it that way

Reply:
Now `items` can be `answer.rrset.items` without needing to check `PY3`, right?

Reply:
🤦